Tests/Unit/Content/StructureResolverTest.php

```diff
@@ -48,6 +50,16 @@ class StructureResolverTest extends TestCase
      */
     private $homepageDocument;
 
+    /**
+     * @var Metadata|ObjectProphecy
+     */
+    private $homepageMetadata;
```

Comment:
i think you do not do anything with this variable right now?

Reply:
i do :) i do - see the `setUp` method